main.py

```diff
@@ -1 +1,23 @@
 #Mad Libs Generator Project
+//Loop back to this point once code finishes
+loop = 1
+while (loop < 9):
+// All the questions that the program asks the user
+    noun = input("Choose a noun: ")
+    plural_noun = input("Choose a plural noun: ")
+    second_noun = input("Choose a noun: ")
+    place = input("Name a place: ")
+    adjective = input("Choose an adjective (Describing word): ")
+    third_noun = input("Choose a noun: ")
+// Displays the story based on the users input
+    print ("------------------------------------------")
+    print ("Be kind to your",noun,"- footed", plural_noun)
+    print ("For a duck may be somebody's", seond_noun,",")
+    print ("Be kind to your",plural_noun,"in",place)
+    print ("Where the weather is always",adjective,".")
+    print ()
```

Comment:
You could also consolidate this by using `\n` at the end of line 17 or at the beginning of the string in line 19 to make a new line.
```suggestion
    print(f"Where the weather is always {adjective}.\n")
```